Pending_updates/com_007_FDI_regulatory_restrictiveness_index/com_007_FDI_regulatory_restrictiveness_index_processing.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+import os
+import sys
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+from zipfile import ZipFile
+import shutil
+import logging
+import glob
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'com_007_FDI_regulatory_restrictiveness_index' #check
```

Comment:
these should be all lower case - we will need to change the python file and overall directory name too, but let's do that after we commit all the suggestions here
```suggestion
dataset_name = 'com_007_fdi_regulatory_restrictiveness_index' #check
```